d2-schemas/src/main/pegasus/com/linkedin/d2/D2Service.pdl

```diff
@@ -96,4 +96,9 @@ record D2Service includes D2ChangeTimeStamps {
    * Configuration of backup requests. Each element describes backup requests configuration for specific operation.
    */
   backupRequests: optional array[BackupRequestsConfiguration]
+
+  /**
+   * The minimum cluster subset size for this service. With a value greater than 0, each client will only send requests to a subset of the hosts in the cluster. A value of -1 disables the feature.
```

Comment:
The document seems missing the case of the value being 0.

Reply:
A value of 0 will be considered as invalid, and will be caught by config validations in d2-c-s.